resources/lib/utils.py

```diff
@@ -227,15 +228,16 @@ def getArt(item, server, widget=False):
         art['tvshow.clearart'] = downloadUtils.getArtwork(item, "Logo", parent=True, server=server)
         art['tvshow.banner'] = downloadUtils.getArtwork(item, "Banner", parent=True, server=server)
         art['tvshow.landscape'] = downloadUtils.getArtwork(item, "Thumb", parent=True, server=server)
+        art['tvshow.fanart'] = downloadUtils.getArtwork(item, "Backdrop", parent=True, server=server)
     elif item_type == "Series":
         art['tvshow.poster'] = downloadUtils.getArtwork(item, "Primary", parent=False, server=server)
         art['tvshow.clearart'] = downloadUtils.getArtwork(item, "Logo", parent=False, server=server)
         art['tvshow.banner'] = downloadUtils.getArtwork(item, "Banner", parent=False, server=server)
         art['tvshow.landscape'] = downloadUtils.getArtwork(item, "Thumb", parent=False, server=server)
+        art['tvshow.fanart'] = downloadUtils.getArtwork(item, "Backdrop", parent=False, server=server)
 
     if item_type == "Episode":
         art['thumb'] = art['thumb'] if art['thumb'] else downloadUtils.getArtwork(item, "Thumb", server=server)
-        art['landscape'] = art['thumb'] if art['thumb'] else downloadUtils.getArtwork(item, "Thumb", parent=True, server=server)
```

Comment:
No I think that should not be set for Episodes as shown here: http://kodi.wiki/view/InfoLabels

The relevant one for landscape is shown below where you can see it should only be set for either a movie or tvshow:


ListItem.Art(landscape) | Returns the 16:9 landscape (if it exists) of the currently selected movie or tv show.
-

Reply:
Probably more appropriate in this case for the episode would be to set:


ListItem.Art(tvshow.landscape) | Returns the tv show landscape (if it exists) of the parent container.
-- | --

Reply:
@sualfred - can you confirm?

Reply:
Yep.

Here is a correct snippet of plugin.video.emby as reference

```
if item_type == "Episode":
            art = {
                'poster': "Series.Primary",
                'tvshow.poster': "Series.Primary",
                'clearart': "Art",
                'tvshow.clearart': "Art",
                'clearlogo': "Logo",
                'tvshow.clearlogo': "Logo",
                'discart': "Disc",
                'fanart_image': "Backdrop",
                'landscape': "Thumb",
                'tvshow.landscape': "Thumb",
                'thumb': "Primary"
            }
        else:
            art = {
                'poster': "Primary",
                'clearart': "Art",
                'clearlogo': "Logo",
                'discart': "Disc",
                'fanart_image': "Backdrop",
                'landscape': "Thumb",
                'thumb': "Primary"
            }
```

You could also take a look at the get_all_artwork function of plugin.video.emby.
https://github.com/MediaBrowser/plugin.video.emby/blob/develop/resources/lib/artwork.py#L485

Reply:
By the way -> Artworks have no fallback rule. The fallbacks are controlled by a skin.

Reply:
I'm a bit confused by this, the snippet shows episode landscape as thumb and skins aren't able to control the fallback that is taking place. The skin could fallback to episode thumb, but fallback to tvshow thumb from episode thumb would not be available to fallback too which is what is happening there.

Reply:
The confusion here is that "Thumb" in Emby terms which is what you see on the right hand side of the snippet actually refers to the Landscape image (kodi terminology). So for the episode you can see the kodi thumb is mapped to the "primary" image from emby. Both landscape and tvshow.landscape are mapped to the parent "Thumb" (which is the landscape image)

Reply:
So actually the code above that I removed should not be deleted it should in fact always set it to the parent thumb:

art['landscape'] = downloadUtils.getArtwork(item, "Thumb", parent=True, server=server)

Reply:
And going one step further the line above should never set the kodi thumb with the parent landscape image:

art['thumb'] = art['thumb'] if art['thumb'] else downloadUtils.getArtwork(item, "Thumb", server=server)

that should be removed altogether as no fallback should be done here

Reply:
So art['tvshow.landscape'] and art['landscape'] are both meant for tvshow landscape in episodes? art['tvshow.landscape'] is already set to parent thumb for episodes.

Reply:
Yes exactly, a bit strange but kodi probably left it like that for the most compatibility with skins... the landscape art should never refer to the thumbnail image as skins may use it to hide spoilers for episodes etc

Reply:
Thank you for clarifying for me, makes way more sense now.

Reply:
Just more useless background history of the values ;) -> 

ListItem.Art(landscape) and Art(tvshow.landscape) aren't official Kodi info values, because no internal Kodi scrapper fills them by default. And with Python you are able to set values like ListItem.Art(myArtworkIsCoolerThanYours) too. But over time, these values have become standard. To get the max compatiblity in all skins it's always better to set the landscape image to both of them. 
The same applies to clearlogo and clearart.

Reply:
Before somebody asks: My snippet from above has ListItem.Art(tvshow.banner) and ListItem.Art(banner) missing.

Reply:
Change the line to what you think it should be.

Reply:
done now with latest commit